AsyncDisplayKit/ASCellNode.mm

```diff
@@ -117,15 +118,18 @@ - (void)layoutDidFinish
   _viewControllerNode.frame = self.bounds;
 }
 
-- (void)__setNeedsLayout
+- (void)displayNodeDidInvalidateSizeOldSize:(CGSize)oldSize
 {
-  CGSize oldSize = self.calculatedSize;
-  [super __setNeedsLayout];
-  
-  //Adding this lock because lock used to be held when this method was called. Not sure if it's necessary for
-  //didRelayoutFromOldSize:toNewSize:
-  ASDN::MutexLocker l(__instanceLock__);
-  [self didRelayoutFromOldSize:oldSize toNewSize:self.calculatedSize];
+  ASSizeRange constrainedSize = ASSizeRangeMake(CGSizeZero, CGSizeMake(CGRectGetWidth(self.bounds), CGFLOAT_MAX));
```

Comment:
Is this condition important with how ASDK currently works?  Just wondering - e.g. do we ever see the interactionDelegate nil when we need it, yet self.bounds is not CGRectZero?

It seems like we should probably default to 0, 0 and max, max - and that it should almost always be overridden by the interactionDelegate